frontend/packages/operator-lifecycle-manager/src/components/operand/utils.spec.ts

```diff
@@ -99,6 +99,24 @@ describe('capabilitiesToUISchema', () => {
     expect(uiSchema['ui:options'].model).toEqual(ServiceAccountModel);
     expect(uiSchema['ui:options'].groupVersionKind).toEqual('ServiceAccount');
   });
+  it('Handles SpecCapability.k8sResourcePrefix with equality-based label queries', () => {
+    const uiSchema = capabilitiesToUISchema([
+      `${SpecCapability.k8sResourcePrefix}ServiceAccount?label!=test,level=production` as SpecCapability,
+    ]);
+    expect(uiSchema['ui:widget']).toEqual('K8sResourceWidget');
+    expect(uiSchema['ui:options'].model).toEqual(ServiceAccountModel);
+    expect(uiSchema['ui:options'].groupVersionKind).toEqual('ServiceAccount');
+    expect(uiSchema['ui:options'].selector).toEqual('label!=test,level=production');
```

Comment:
These test cases need to be updated. A Selector object is expected now:
```suggestion
    expect(uiSchema['ui:options'].selector).toEqual(
      selectorFromString('label!=test,level=production')
    );
```